Assets/RedBlueGames/BulkRename/Editor/BulkRenamerWindow.cs

```diff
@@ -211,64 +210,33 @@ private void OnGUI()
             EditorGUILayout.EndScrollView();
         }
 
-        private string[] GetNamesFromSelections()
+        private void SortObjectsToRename()
         {
-
-            List<string> allAssetNames = new List<string>();
-            List<Object> allSortedGameObectsNames = new List<Object>();
-
-            int namesCount = this.objectsToRename.Count;
-            var names = new string[namesCount];
-            bool isItAsset = AssetDatabase.GetAssetPath(objectsToRename[0]) != "";
```

Comment:
I haven't seen that issue with the assets not showing but the screenshot is super helpful, thanks. I'll have to dig into that.